app/liquid_tags/user_subscription_tag.rb

```diff
@@ -11,328 +11,331 @@ def self.user_authorization_method_name
     :super_admin,
   ].freeze
 
-  SCRIPT = <<~JAVASCRIPT.freeze
-    var subscribeBtn = document.getElementById('subscribe-btn');
-
-    function isUserSignedIn() {
-      return document.head.querySelector('meta[name="user-signed-in"][content="true"]') !== null;
-    }
-
-    // Hiding/showing elements
-    // If clearSubscribeButton is false, we will not clear out the subscription-signed-in area,
-    // which will allow users to re-submit their subscription if they see any error messages.
-    // ***************************************
-    function clearSubscriptionArea({ clearSubscribeButton = true } = {}) {
-      if (!clearSubscribeButton) {
-        // Allow users to try submitting again if they see an error.
-        hideSubscriptionSignedIn();
-      }
+  def self.script
```

Comment:
A bit surprised by this change.  My assumption is that the javascript didn't change (aside from each line being indented once more).

Could you talk me through this change?

Reply:
It is correct. As I get from previous CI, the `I18n` module initialization (at least, loading of locale YAMLs) seems to come later than the constant initialization, so when you try to insert the output of I18n to a constant as in my original attempt, it only returns `translation missing: ...`.

Of course, a better solution would be appreciated.